frontend/app/routes/protected/multi-channel/search-sin.tsx

```diff
@@ -54,17 +54,19 @@ export async function action({ context, request }: Route.ActionArgs) {
     }
 
     case 'search': {
-      const sinCaseService = getSinCaseService();
-      const sinCases = await sinCaseService.getSinCases();
+      // TODO: pull caseId from search param when ready
+      const caseId = '';
+      const sinSearchService = getSinSearchService();
+      const searchResults = (await sinSearchService.getSearchResults(caseId)) ?? [];
```

Comment:
`getSearchResults(..)` should be updated to always return a value.. when no results are found, it should return `[]`.